dist/workerpool.js

```diff
@@ -978,7 +978,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	        me.processing[id].resolver.reject(error);
 	      }
 	    }
-	    me.processing = {};
+	    me.processing = Object.create(null);
```

Comment:
`me.processing` is ulltimately a growing/shrinking dictionary object, using `Object.create(null)` both ensures the prototype isn't polluted, and also gives a good hint to the JIT. This gets most JITs to skip trying to figure out what shape the object is, and rather go straight to dictionary mode.